src/common/components/InputField/Input/Input.tsx

```diff
@@ -0,0 +1,44 @@
+import { ChangeEvent } from "react";
+import classNames from "classnames/bind";
+import styles from "./Input.module.scss";
+
+const cn = classNames.bind(styles);
+
+interface InputProps {
+  type: "text" | "email" | "password" | "dropdown" | "search";
+  size?: "sm" | "md" | "full";
+  placeholder?: string;
+  name: string;
+  color?: "white" | "gray";
+  disabled?: true | false;
```

Comment:
이렇게 표현해주는게 더 좋을 것 같네요~
```suggestion
  disabled?: boolean;
```